### PR TITLE
WS-3203: add value per click fields

### PR DESCRIPTION
**Pull request by srwang**
Per https://2u-internal.atlassian.net/browse/WS-3203, we are adding the ability to edit value per click on a course. Value per click will be used in Algolia's search rankings. 

The fields added are `in_year_value.per_click_usa` and `in_year_value.per_click_international`. These fields should only be visible and able to be edited by administrators.

### CODE REVIEW

**Review by ansabgillani — APPROVED** [review]

File: src/components/EditCoursePage/EditCoursePage.test.jsx [hunk 1]
```diff
@@ -957,6 +957,8 @@ describe('EditCoursePage', () => {
       const inYearValue = {
         per_lead_usa: '100',
         per_lead_international: null,
+        per_click_usa: null,
```
Comment (AliAdnanSohail) [inline src/components/EditCoursePage/EditCoursePage.test.jsx:960]: nit: move `per_click_usa` and `per_click_internation` above the `per_lead_usa `

File: src/components/EditCoursePage/EditCoursePage.test.jsx [hunk 2]
```diff
@@ -966,6 +968,8 @@ describe('EditCoursePage', () => {
         in_year_value: {
           per_lead_usa: 100,
           per_lead_international: 0,
+          per_click_usa: 0,
```
Comment (AliAdnanSohail) [inline src/components/EditCoursePage/EditCoursePage.test.jsx:971]: nit: move per_click_usa and per_click_internation above the per_lead_usa

**Review by AliAdnanSohail — APPROVED** [review]
Looks good to me. Just 2 small nits

**Comment by ansabgillani** [comment]
Please address Ali's comments and squash. @srwang

**Pull request merged**